<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <item>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tabServer">
       <attribute name="title">
        <string>server</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="leSourceFile"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbSelectSource">
             <property name="text">
              <string>select</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbStartThread">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbStopThread">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbDirectBF">
             <property name="text">
              <string>DirectBF</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetFilter" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="leFilter"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbSetFilter">
             <property name="text">
              <string>Filter</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widgetCmd" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>count:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbCount">
             <property name="maximum">
              <number>99999</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>ignore:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbIgnore">
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>rate_ctl:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leRateCtl">
             <property name="text">
              <string>0x2C</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>phase:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lePhase">
             <property name="text">
              <string>0,1,2,3,4,5,6,7,8,9,A,B</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>data</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTableWidget" name="twData">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>200</number>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>phase</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>idx</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>data</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_5" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QPushButton" name="pbClearData">
                 <property name="text">
                  <string>clear</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pbSaveData">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabLog">
           <attribute name="title">
            <string>log</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QTextEdit" name="log"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAgent">
       <attribute name="title">
        <string>agent</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Server IP:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leServerIP">
             <property name="inputMask">
              <string>000.000.000.000;_</string>
             </property>
             <property name="text">
              <string>127.0.0.1</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Port:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbPort">
             <property name="minimum">
              <number>10000</number>
             </property>
             <property name="maximum">
              <number>65534</number>
             </property>
             <property name="value">
              <number>38010</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbBind">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>rate_ctl:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leRateCtlPath">
             <property name="text">
              <string>/proc/net/rtl88x2bu/wlan3/rate_ctl</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbRateCtl">
             <item>
              <property name="text">
               <string>0x2C</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbTestRateCtl">
             <property name="text">
              <string>Test rate_ctl</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>pathb_phase:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lePathbPhasePath">
             <property name="text">
              <string>/proc/net/rtl88x2bu/wlan3/pathb_phase</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbPhase">
             <item>
              <property name="text">
               <string>0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>A</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>B</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbTestPathbPhase">
             <property name="text">
              <string>Test pathb_phase</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_6" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="1">
            <widget class="QLineEdit" name="leRateSearchResult">
             <property name="text">
              <string>/proc/net/rtl88x2bu/wlan3/rate_search_result</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="lbRateSearch">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>rate_search_result:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="leRateSearch">
             <property name="text">
              <string>/proc/net/rtl88x2bu/wlan3/rate_search</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>rate_search:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pbSingelTrigger">
             <property name="text">
              <string>Trigger</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="pbGetRate">
             <property name="text">
              <string>Get</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QTextEdit" name="teRateSearchResult"/>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="pbGetRateSerachResult">
             <property name="text">
              <string>Get Result</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAtt">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QWidget" name="widget_8" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>IP:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leIP">
             <property name="text">
              <string>192.168.110.239</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Account:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leUserName">
             <property name="text">
              <string>tld1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Password:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lePassword">
             <property name="text">
              <string>123456</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_7" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLineEdit" name="leRSSI">
             <property name="text">
              <string>/proc/net/rtl88x2bu/wlan1/alpha_rx_signa</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbEVPathSignal">
             <property name="text">
              <string>ev path signal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbSaveEVPathSignal">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="twAtt">
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Att</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Start</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Stop</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Step</string>
           </property>
          </column>
          <item row="0" column="1">
           <property name="text">
            <string>40</string>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string>100</string>
           </property>
          </item>
          <item row="0" column="3">
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string>40</string>
           </property>
          </item>
          <item row="1" column="2">
           <property name="text">
            <string>100</string>
           </property>
          </item>
          <item row="1" column="3">
           <property name="text">
            <string>1</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="twSignal">
          <column>
           <property name="text">
            <string>Att1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Att2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>RSSI1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>RSSI2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>SNR1</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>SNR2</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionagent">
   <property name="text">
    <string>agent</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
